File: Cpp/Minimum_Number_of_Increments_on_Subarrays_to_Form_a_Target_Array.cpp
class Solution {
public:
    int minNumberOperations(vector<int>& target) {
        int op = target[0];

        for (int i = 1; i < target.size(); i++) {
            if (target[i] > target[i - 1]) {
                op += target[i] - target[i - 1];
            }
        }

        return op;
    }
};

File: Rust/Minimum_Number_of_Increments_on_Subarrays_to_Form_a_Target_Array.rs
impl Solution {
    pub fn min_number_operations(target: Vec<i32>) -> i32 {
        let mut op = target[0];
        for i in 1..target.len() {
            if target[i] > target[i - 1] {
                op += target[i] - target[i - 1];
            }
        }
        op
    }
}
